src/lib/sensor/calibration/Accelerometer.cpp

```diff
@@ -154,93 +159,65 @@ bool Accelerometer::set_calibration_index(int calibration_index)
 
 void Accelerometer::ParametersUpdate()
 {
-	if (_device_id == 0) {
+	if (device_id() == 0) {
 		return;
 	}
 
-	_calibration_index = FindCurrentCalibrationIndex(SensorString(), _device_id);
+	_calibration_index = FindCurrentCalibrationIndex(SensorString(), device_id());
 
 	if (_calibration_index == -1) {
 		// no saved calibration available
 		Reset();
 
 	} else {
 		ParametersLoad();
+
+		if (calibrated() && !_configuration.configured()) {
+			_configuration.ParametersSave(calibration_index());
```

Comment:
This was kind of a lazy way to establish the sensor configuration slots (so a regular user can find them). 

It might be better to do it more explicitly in a different area.